src/clp_ffi_py/ir/error_messages.hpp

```diff
@@ -3,6 +3,8 @@
 
 namespace clp_ffi_py::ir {
 constexpr char const* cDecoderBufferOverflowError = "DecoderBuffer internal read buffer overflows.";
+constexpr char const* cDecoderIncompleteIRError = "The IR stream is incomplete.";
+constexpr char const* cDecoderErrorCodeTemplate = "IR decoding method failed with error code: %d.";
```

Comment:
```suggestion
constexpr char const* cDecoderErrorCodeFormatStr = "IR decoding method failed with error code: %d.";
```

Optional change:
I'd say calling this a format string is more common than a template.